src/main/resources/assets/tinkersurvival/lang/zh_cn.json

```
@@ -11,107 +11,107 @@
-    "modifier.insulated_armor.description": "在装甲零件上添加 +1 加热调节剂。不可堆叠。",
+    "modifier.insulated_armor": "保暖",
+    "modifier.insulated_armor.description": "在盔甲上添加+1加热强化。不可堆叠。",
     "modifier.chilling_armor": "降温",
-    "modifier.chilling_armor.description": "在装甲零件上添加 -1 制冷调节器。不可堆叠。",
-    "modifier.tinkersurvival.mana": "Mana",
-    "modifier.tinkersurvival.mana.description": "如果存在提供魔力的物品，则修复背包中的物品。",
+    "modifier.chilling_armor.description": "在盔甲上添加-1制冷强化。不可堆叠。",
+    "modifier.tinkersurvival.mana": "魔力",
+    "modifier.tinkersurvival.mana.description": "如果提供魔力，物品栏中的物品将使用魔力修复耐久度。",
     "tools": "工具",
-    "MOD_TOOL_WHITELIST.tooltip": "依然能使用的模组工具列表。而其他模组的工具将变成“湿面条”。",
-    "TOOLS_WHITELIST.tooltip": "仍然能使用的个别工具列表。格式 [tooltype-modid:item]",
-    "armor": "护甲",
-    "MOD_ARMOR_WHITELIST.tooltip": "仍然能合成和使用的模组盔甲列表。而其他模组的盔甲不能装备或合成。",
-    "ARMOR_WHITELIST.tooltip": "仍然能使用的个别盔甲列表。格式 [modid:item]",
-    "ARMOR_TEMP_MODIFIERS.tooltip": "所有有温度调节功能的盔甲列表。格式 [modid:item=modifier]",
+    "MOD_TOOL_WHITELIST.tooltip": "列表中模组的工具将始终有效并且可以制作。而其他模组的工具则会变得像湿面条一样。",
+    "TOOLS_WHITELIST.tooltip": "列表中始终有效的单个工具。格式为tooltype-modid:item",
+    "armor": "盔甲",
+    "MOD_ARMOR_WHITELIST.tooltip": "列表中模组的盔甲将始终有效并且可以制作。未在列表中的模组的盔甲将无法使用并无法制作。",
+    "ARMOR_WHITELIST.tooltip": "列表中始终有效的单个盔甲。格式为modid:item",
+    "ARMOR_TEMP_MODIFIERS.tooltip": "所有有温度调节强化的盔甲列表。格式为modid:item=modifier",
     "INSULATED_MODIFIER.tooltip": "匠魂盔甲羊毛的保温系数值。",
     "CHILLING_MODIFIER.tooltip": "匠魂盔甲冻结史莱姆的降温系数值。",
-    "balance": "平衡",
-    "ENABLE_ROCKGEN.tooltip": "开启生成在主世界的石头堆。默认为开启",
-    "ROCKGEN_CHANCE.tooltip": "石头在主世界的生成概率。默认 1.0D (= 100%)",
-    "ENABLE_ROCK_FROM_DIRT.tooltip": "开启岩石能在打破泥土后获得。默认为不开启",
-    "ROCK_FROM_DIRT_CHANCE.tooltip": "石头从空手打破泥土中的获得几率。默认 0.4 (= 40%)",
-    "FLINT_CHANCE.tooltip": "成功打落燧石的几率。默认 60% - 0.6D",
-    "GRASS_FIBER_CHANCE.tooltip": "高草丛掉落植物纤维的概率。默认 60% - 0.6D。刀子的效率提高 40%。",
-    "HEAL_RATE.tooltip": "绷带的治疗率。粗制绷带的效果降低 50%。",
-    "SLOW_DOWN_MULTIPLIER.tooltip": "调整使用错误工具时的缓慢效果。默认 1.0)",
-    "features": "游戏特点",
-    "NO_GRIEFING.tooltip": "禁止末影人破坏地形，wtf Notch。",
-    "GRIEFING_WHITELIST.tooltip": "好吧，我喜欢末影人，让他们捡起这些东西。如果 NO_GRIEFING = true",
-    "NO_SLEEPING.tooltip": "在晚上，我哭着想去睡觉......",
-    "SLOW_DOWN_DIRT_PUNCHING.tooltip": "包括泥土空手挖掘速度也变慢。默认为开启",
-    "ENABLE_SAW.tooltip": "锯子能够制作木板。默认为开启",
-    "FORCE_SAW_FOR_PLANKS.tooltip": "从游戏中去掉木板和木棍的配方，只有锯子才能够制作它们。默认为开启",
+    "balance": "平衡性",
+    "ENABLE_ROCKGEN.tooltip": "岩石堆能生成在地表。默认为true",
+    "ROCKGEN_CHANCE.tooltip": "岩石堆生成在地表的概率。默认为1.0D（=100%）",
+    "ENABLE_ROCK_FROM_DIRT.tooltip": "挖掘泥土能掉落小型岩石。（默认：false)",
+    "ROCK_FROM_DIRT_CHANCE.tooltip": "空手挖掘泥土掉落小型岩石的概率。默认为0.4（=40%）",
+    "FLINT_CHANCE.tooltip": "敲击获得燧石碎片的成功概率。默认为60%-0.6D",
+    "GRASS_FIBER_CHANCE.tooltip": "高草掉落植物纤维的概率。默认为60%-0.6D。使用小刀的概率再多40%。",
+    "HEAL_RATE.tooltip": "调整绷带的生命值恢复速度。粗制绷带的效果为50%。",
+    "SLOW_DOWN_MULTIPLIER.tooltip": "用于调整错误使用工具时的挖掘速度削减。（默认为1.0)",
+    "features": "游戏特性",
+    "NO_GRIEFING.tooltip": "别让末影人再搞破坏了，什么？！Notch。",
```

```suggestion
    "NO_GRIEFING.tooltip": "禁止末影人破坏事物，Notch你在做啥。",
```
This change is only a suggestion
I suppose the first half of sentence is an independent statement from the other half, possibly wrong, no need to change that part if finding it strange to read after editing
The second part is changed to a somewhat question(lit. what are you doing Notch.), maybe it fits the original text better

---

I have no idea of this, so I just improve the orginal translation. Maybe we should discuss this